src/salesforce_uploader.js

```diff
@@ -12,10 +12,17 @@ export function handler (input:?any, context:?any, callback:Function) {
     callback(new NamedError('inputError', 'Input did not contain filename'))
     return
   }
+
+  if (input.preventUpload !== null && input.preventUpload) {
+    callback(new NamedError('preventUploadError', 'Upload to salesforce disabled.'))
```

Comment:
maybe we could skip the last step using a choice state in the state machine:
http://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-choice-state.html